metricbeat/module/system/_meta/kibana/default/dashboard/Metricbeat-containers-overview.json

```diff
@@ -13,7 +13,7 @@
       },
       "id": "Container-CPU-usage",
       "type": "visualization",
-      "version": 2
+      "version": 1
```

Comment:
What do these version changes mean?

Reply:
I think they can be ignored, it seems that the version is incremented everytime you save a dashboard in Kibana, but I don't think they have a relevance beyond that.

Reply:
yes, we can ignore that for now.